src/core/text/TextStyle.js

```diff
@@ -232,6 +235,19 @@ export default class TextStyle
         }
     }
 
+    get fillGradientStops()
+    {
+        return this._fillGradientStops;
+    }
+    set fillGradientStops(fillGradientStops)
+    {
+        if (this._fillGradientStops !== fillGradientStops)
```

Comment:
While it's simple, I think there might be a better comparison than strict check. Since it is an array, might be useful if you compare the actual array values. For instance:

```js
text.style.fillGradientStop = [0, 0.2, 0.4];
text.style.fillGradientStop = [0, 0.2, 0.4]; // shouldn't re-render
```